docs/release_notes/v0.16.0.md

```diff
@@ -0,0 +1,35 @@
+# New in 0.16.0 (2022-04-08)
+
+In this release, we have refactored our destVI model:
```

Comment:
see a previous release note for how to reference a model (will automatically hyperlink it to the docstring). I'd use this reference notation each time you refer to a model by name